src/app/Fake.Core.Variables/Variables.fs

```diff
@@ -0,0 +1,59 @@
+﻿/// This module contains helpers for managing build time variables
+module Fake.Core.Variables
```

Comment:
Should we add `[<RequireQualifiedAccess>]`?

Reply:
Agreed